README.md

```diff
@@ -691,10 +696,35 @@ The following protocol specification features are not currently implemented:
   recorded in issue [100](https://github.com/yahoo/proxy-verifier/issues/100).
 * Only TCP is supported. There have been recent discussions about adding
   HTTP/3 support, which is over UDP, but work for that has not yet started.
+* The PROXY protocol support is limited to the `PROXY` command via TCP over IPv4 or IPv6. Therefore,
+  * No support for `AF_UNIX` socket address as either source and destination address.
+  * No support for UDP transport type.
+  * No support for `LOCAL` command(in v2 header).
 
 If there is no `protocol` node specified, then Proxy Verifier will default to
 establishing an HTTP/1 connection over TCP (no TLS).
 
+#### PROXY protocol support
+Generally speaking, a server sitting downstream from a proxy does not have
+visibility into the client's network socket information that lies behind the
+proxy. PROXY Protocol provides a mechanism to provide visibility for this. PROXY
```

Comment:
Sorry, Zhengxi. My fault. There are two "provides" in this sentence. Maybe:

```YAML
PROXY Protocol is a mechanism to provide visibility for this.
```

Reply:
Fixed. Thank you!